ext/js/dictionary/dictionary-importer.js

```diff
@@ -815,8 +815,19 @@ export class DictionaryImporter {
         const results = [];
         for (const file of files) {
             const content = await this._getData(file, new TextWriter());
-            /** @type {unknown} */
-            const entries = parseJson(content);
+            let entries;
+
+            try {
+                /** @type {unknown} */
+                entries = parseJson(content);
+            }
+            catch (error) {
+                if (error instanceof Error) {
+                    let newError = new Error(error.message + `. Dictionary has invalid data in '${file.filename}'`);
+                    console.error(newError);
```

Comment:
You should use `log.error` instead. `log` is a wrapper around `console`.

Reply:
Thank you. Can you elaborate on what makes log.error more appropriate?

Reply:
You can check out https://github.com/themoeway/yomitan/blob/master/ext/js/core/log.js#L74. It basically formats the log with info for the extension.